### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by mwarkentin** [hubot-scripts/hubot-http-status#1]
All of my requested statuses error out - I'm guessing the structure of the page might've changed and broken the scraping code:

```
hubot http status 422
Hubot> HTTP status code '422' doesn't exist. Ironically, this would be HTTP Error 404.
Hubot> hubot http status 416
Hubot> HTTP status code '416' doesn't exist. Ironically, this would be HTTP Error 404.
Hubot> hubot http status 419
Hubot> HTTP status code '419' doesn't exist. Ironically, this would be HTTP Error 404.
Hubot> hubot http status 404
Hubot> HTTP status code '404' doesn't exist. Ironically, this would be HTTP Error 404.
```

**Comment by chrisvfritz** [comment]
I can confirm. Just did a little digging and `$('#200').parent().text()` on the page does indeed return `"200 OK"`, so possibly something wrong with cheerio or the returned body being wrong?

**Comment by chrisvfritz** [comment]
Just fixed this in #2.
